Esteban-Salas-Posada/tarea-11/routes/src/components/Navbar.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import './navbar.css' 
+import { NavLink } from 'react-router-dom'
+
+export default function Navbar() {
+  return (
+      <>
+        <ul>
+            <li> <NavLink className={({isActive})=> isActive ? 'active' : 'link' } to='/task1'>Task 1</NavLink> </li>
```

Comment:
@esteban-salas , this is a bad approach , links should be generated dynamically from an Array